free-2.md

```diff
@@ -46,6 +55,7 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are HDLs like SystemVerilog HDL, but they are what this DSL is trying to help replace. I did find some more interesting HDLs, though. For example, a few people created [a way to make hardware designs in Haskell](http://www.cse.chalmers.se/edu/year/2012/course/TDA956/Papers/Lava98.pdf) of all languages. While this is interesting, I'm not sure how many people want to work with Haskell's syntax while also trying to design hardware. Although, you can use Haskell with an interpreter, which is nice. I think it would probably be better to use syntax that is more appropriate for the task at hand and Haskell wasn't exactly meant to be pushed to be used in this strange way.
```

Comment:
This is a good insight - Haskell is almost an esoteric language for people who aren't familiar with functional programming so it seems like a poor language to write an internal DSL in unless you can heavily subvert it.